subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -953,6 +955,28 @@ static void mfy_cis_offset_get(void *param)
 			cig_remainder_us + cig->sync_delay -
 			acl_remainder_us - cis->sync_delay;
 
+	/* Calculate instant latency */
+	latency_acl = cis->central.instant - ull_conn_event_counter(conn);
+	elapsed_acl_us = latency_acl * conn->lll.interval * CONN_INT_UNIT_US;
```

Comment:
Is 32-bit always enough to cover the result of `latency_acl * conn->lll.interval * CONN_INT_UNIT_US`?

Reply:
Good question, will rework on this based on spec maximums

Reply:
added code comments.